README.md

```diff
@@ -16,12 +16,17 @@ when the project is compiled.
 
 In a few integration tests we use [Aiven's JDBC connector](https://github.com/aiven/jdbc-connector-for-apache-kafka).
 Since it's not available in public Maven repositories, it needs to be compiled and installed to a local Maven repository,
-so it can be used. For convenience, we provide a prebuilt JAR (in the `libs` directory).
+so it can be used. To do so:
 
-1. Run `mvn install:install-file -Dfile=libs/jdbc-connector-for-apache-kafka-6.7.0.jar -DgroupId=io.aiven -DartifactId=jdbc-connector-for-apache-kafka -Dversion=6.7.0 -Dpackaging=jar`
-2. Run `mvn clean compile` (so that the needed code is generated)
-3. Some IDEs may not automatically use the generated sources. If that's the case, you need to change the project settings in your IDE to include the generated source. In IntelliJ, for example, you do that by going
-to File > Project structure > Project Settings > Modules. Then, right-click on `target/generated-source` and select "Sources".
+1. Clone the repository: `git clone git@github.com:Aiven-Open/jdbc-connector-for-apache-kafka.git`
```

Comment:
```suggestion
1. Clone the Aiven JDBC connector repository: `git clone git@github.com:Aiven-Open/jdbc-connector-for-apache-kafka.git`
```

Reply:
Also, maybe consider changing all of this into a script that can be copied, so we don't have to run each command separately, like:
```
# clone the Aiven JDBC connector repository and cd into the directory
git clone git@github.com:Aiven-Open/jdbc-connector-for-apache-kafka.git
cd jdbc-connector-for-apache-kafka

# check out tag v6.8.0
git checkout v6.8.0

# build and publish to local Maven repository
./gradlew clean build publishToMavenLocal
```

Reply:
Sounds good!